src/lua/widget/combobox.c

```diff
@@ -68,8 +68,6 @@ static int combobox_numindex(lua_State*L)
     return 0;
   }
   if(key <= 0 || key > length) {
-    return luaL_error(L,"Invalid index for combo box : %d\n",key);
-  } else if (key > length) {
```

Comment:
wait... I don't see how this is related to ipairs, 

yes you are supposed to be able to iterate with ipairs, do you know why it didn't work , did ipair try to access out of bound element ?

maybe it would be enough to return nil for key == length+1 and an error for other access... 

I don't want to allow access to negative index (not sure exactly how lua reacts to that) and i'm a bit worried about accessing random beyond-end indices. If you set a random non-existing index, we would need to "fill up" the widget with the missing elements ?